2022-01-05 21:54:55,081 INFO    MainThread:445827 [wandb_setup.py:_flush():71] setting env: {}
2022-01-05 21:54:55,081 INFO    MainThread:445827 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-05 21:54:55,082 INFO    MainThread:445827 [wandb_init.py:_log_setup():357] Logging user logs to /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220105_215455-ht6emxqe/logs/debug.log
2022-01-05 21:54:55,082 INFO    MainThread:445827 [wandb_init.py:_log_setup():358] Logging internal logs to /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220105_215455-ht6emxqe/logs/debug-internal.log
2022-01-05 21:54:55,082 INFO    MainThread:445827 [wandb_init.py:init():390] calling init triggers
2022-01-05 21:54:55,082 INFO    MainThread:445827 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {'width': 25, 'height': 25, 'num_agents': 1, 'max_num_cities': 2, 'seed': 1, 'grid_mode': False, 'max_rails_between_cities': 2, 'max_rails_in_city': 4, 'malfunction_rate': 0, 'min_duration': 20, 'max_duration': 50, 'observation_builder': 'GraphObsForRailEnv', 'predictor': 'ShortestPathPredictorForRailEnv', 'view_semiwidth': 7, 'view_height': 30, 'offset': 25, 'observation_depth': 3, 'num_episodes': 200, 'start_epoch': 0, 'max_steps': 300, 'eps': 1, 'debug_print': False, 'load_memory': False, 'evaluation_episodes': 3, 'render': False, 'evaluation_interval': 10, 'save_model_interval': 50, 'start_lr_decay': 150, 'eps_decay': 0.999, 'learning_rate': 0.02, 'learning_rate_decay': 0.5, 'learning_rate_decay_policy': 0.5, 'run_title': 'epsilon fixed', 'wandb_project_name': 'flatland', 'model_path': 'test_results/', 'model_name': '', 'gat_layers': 3, 'flow': 'source_to_target', 'resume_weights': True, 'batch_norm': False, 'dropout_rate': 0.8, 'attention_heads': 4, 'batch_size': 128, 'use_stop_action': False, 'done_reward': 0, 'deadlock_reward': -1000, 'reward_scaling': 0.1}
2022-01-05 21:54:55,082 INFO    MainThread:445827 [wandb_init.py:init():435] starting backend
2022-01-05 21:54:55,082 INFO    MainThread:445827 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-05 21:54:55,091 INFO    MainThread:445827 [backend.py:ensure_launched():199] starting backend process...
2022-01-05 21:54:55,098 INFO    MainThread:445827 [backend.py:ensure_launched():204] started backend process with pid: 445892
2022-01-05 21:54:55,099 INFO    MainThread:445827 [wandb_init.py:init():444] backend started and connected
2022-01-05 21:54:55,101 INFO    MainThread:445827 [wandb_init.py:init():503] updated telemetry
2022-01-05 21:54:55,120 INFO    MainThread:445827 [wandb_init.py:init():533] communicating current version
2022-01-05 21:54:56,338 INFO    MainThread:445827 [wandb_init.py:init():538] got version response upgrade_message: "wandb version 0.12.9 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-01-05 21:54:56,338 INFO    MainThread:445827 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2022-01-05 21:54:57,247 INFO    MainThread:445827 [wandb_init.py:init():576] starting run threads in backend
2022-01-05 21:54:57,381 WARNING MainThread:445827 [wandb_init.py:init():891] interrupted
Traceback (most recent call last):
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 870, in init
    run = wi.init()
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 581, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 216, in communicate_run_start
    result = self._communicate_run_start(run_start)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/interface/interface_queue.py", line 446, in _communicate_run_start
    result = self._communicate(rec)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/interface/interface_queue.py", line 232, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/interface/router.py", line 32, in get
    is_set = self._object_ready.wait(timeout)
  File "/home/runnphoenix/anaconda3/lib/python3.8/threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "/home/runnphoenix/anaconda3/lib/python3.8/threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
